tokio/src/runtime/metrics/runtime.rs

```diff
@@ -446,4 +446,14 @@ impl RuntimeMetrics {
     pub fn worker_local_queue_depth(&self, worker: usize) -> usize {
         self.handle.spawner.worker_local_queue_depth(worker)
     }
+
+    pub fn io_driver_fd_count(&self) -> Option<u64> {
```

Comment:
These return options do we want to do this if the io driver has dropped? Or should we just panic?

Reply:
Default to zero. If it is `None`, the runtime is shutdown so there are no more registered FDs.

Reply:
Oh, I see the issue is that the counter should always increment...yeah, we don't want `Option` in the final API...

Reply:
I would start w/ `unwrap_or(0)` for now, but track a TODO to fix before stabilizing the API.